pjmedia/src/pjmedia/transport_srtp_dtls.c

```diff
@@ -1247,12 +1249,31 @@ static pj_status_t dtls_encode_sdp( pjmedia_transport *tp,
 		    (ds->setup==DTLS_SETUP_ACTIVE? &ID_ACTIVE:&ID_PASSIVE));
 	pjmedia_sdp_media_add_attr(m_loc, a);
 
+        if (last_setup != DTLS_SETUP_UNKNOWN && sdp_remote) {
+            pj_sockaddr rem_rtp;
+            pj_sockaddr rem_rtcp;
+
+            pj_bzero(&rem_rtp, sizeof(rem_rtp));
+            pj_bzero(&rem_rtcp, sizeof(rem_rtcp));
```

Comment:
unnecessary? `pj_sockaddr_init()` will be used in `get_rem_addrs()`

Reply:
ok